src/components/Table/hoc/withTableSelection/withTableSelection.scss

```diff
@@ -7,13 +7,19 @@
     }
 
     &__selection-checkbox {
+        display: flex;
+        align-items: center;
```

Comment:
Let's add an extra container for Checkbox and not put layout properties to the Checkbox itself, we can move positioning properties there as well

Reply:
But we have to expand the checkbox label tag over the entire cell to have an ability clicking in any cell place to toggle selection. So it seems like if i will wrap that Checkbox component and center it using this wrapper - this label tag(that we need to being extended) will not take all cell place because of "padding" that centers that checkbox component